content-repo/gen_top_contrib.py

```diff
@@ -106,7 +106,10 @@ def github_pagination_prs(url: str, params: dict, res) -> list:
         links = link.split(',')
         last_page = links[-1]
         last_page_link = last_page[last_page.find("<")+1:last_page.find(">")]
-        last_pr_number = PAGE_NUMBER_REGEX.search(last_page_link)[0]
+        try:
+            last_pr_number = PAGE_NUMBER_REGEX.search(last_page_link)[0]
+        except IndexError:
+            print(f"Couldn't find the last page number. \n Last page link - {last_page_link}")
```

Comment:
You don't want to continue in this case
```suggestion
            print(f"Error: Couldn't find the last page number. \n Last page link - {last_page_link}")
            return
```